app/src/main/java/com/rockspin/androiddevtest/APICommThread.java

```diff
@@ -0,0 +1,33 @@
+package com.rockspin.androiddevtest;
+
+import android.content.Intent;
+
+import java.util.List;
+
+import retrofit2.Response;
+
+/**
+ * Created by SG on 08.01.2017 г..
+ */
+public class APICommThread extends Thread
+{
+    @Override
```

Comment:
I try to keep as much as possible with what the underlying platform gives me as libraries, to avoid adding extra overhead in the app. Also for me relying on third party libs for something simple as threads is not needed. During my 8 years of professional programming I had several major bugs due to third party libs.